posts/inside-rust/2020-09-11-stabilizing-intra-doc-links.md

```diff
@@ -0,0 +1,215 @@
+layout: post
+title: "Intra-doc links close to stabilization"
+author: Manish Goregaokar and Joshua Nelson
+team: the rustdoc team <https://www.rust-lang.org/governance/teams/dev-tools#rustdoc>
+---
+
+# Intra-doc links close to stabilization
+
+We're excited to share that
```

Comment:
Isn't punctuation missing here?

Reply:
oh it seems like some of the text got dropped @jyn514

Reply:
Hmm, that's what I see on the HackMD:

![image](https://user-images.githubusercontent.com/23638587/93004545-ed258980-f515-11ea-96af-c42709e8855d.png)

Did you mean to write more?

Reply:
Alright, I must have messed up. This should be something like "We're excited to share that intra-doc links are stabilizing soon!"